app/channel-settings/client/startup/messageTypes.js

```diff
@@ -64,4 +64,15 @@ Meteor.startup(function() {
 			};
 		},
 	});
+
+	MessageTypes.registerType({
```

Comment:
there is a `room_changed_avatar` registered already in line 32
https://github.com/RocketChat/Rocket.Chat/blob/142b3171170b321f55e63f63b2224a0d6cabac45/app/channel-settings/client/startup/messageTypes.js#L32-L41